utils/combineSortedArray.js

```diff
@@ -0,0 +1,20 @@
+const deepCopy = require('./deepCopy');
```

Comment:
The file was added specifically to this commit. Can you explain?

Reply:
The `combineSortedArray` function was defined in the `mamUtils.js`, it looks like a generic utility function so I move it out of the `mamUtils.js`.

Reply:
The `combineSortedArray` function may not be able to use in other situation. So I think just keep it in `mamUtils.js` will be nice.